src/test/run-pass/yk_swt/collect-trace.rs

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 King's College London.
+// Created by the Software Development Team <http://soft-dev.org/>.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![feature(yk_swt)]
+#![feature(libc)]
+
+extern crate core;
+extern crate libc;
+
+use core::yk_swt::{start_tracing, stop_tracing};
+
+pub fn main() {
+    start_tracing();
+    let _ = work();
```

Comment:
Why the `let _ =`?

Reply:
I'm about 98% certain that assigning to the dummy variable won't affect what the compiler does in that way. You might want the `black_box` function?

Reply:
OK, I'll look into that.